gcloud/_testing.py

```diff
@@ -20,6 +20,8 @@ class _Monkey(object):
 
     def __init__(self, module, **kw):
         self.module = module
+        if len(kw) == 0:  # pragma: NO COVER
+            raise ValueError('_Monkey was used with nothing to monkey-patch')
```

Comment:
@tseaver I can ditch this if you like. I added it since I noticed there was a `_Monkey(MUT)` in `test_service_account_via_json_key` before I added in the `_OpenSSLCrypto` stub